src/encoded/static/components/summary.js

```diff
@@ -70,6 +40,21 @@ function generateStatusData(buckets, labels) {
     return statusData;
 }
 
+// Data field for organism
+// We will display different facets depending on the selected organism
+const organismField = 'replicates.library.biosample.donor.organism.scientific_name';
+// Query for systems slim
+const systemsField = 'biosample_ontology.system_slims';
+// Query for organ slim
+const organField = 'biosample_ontology.organ_slims';
```

Comment:
Could you import `systemsField` and `organField` from body_map.js so both places don’t need to change if they ever need changing? You could still make `BodyMap` the default export.